sdk/python/tests/compiler/compiler_tests.py

```diff
@@ -364,7 +364,7 @@ def test_py_param_substitutions(self):
   def test_type_checking_with_consistent_types(self):
     """Test type check pipeline parameters against component metadata."""
     @component
-    def a_op(field_m: {'GCSPath': {'path_type': 'file', 'file_type':'tsv'}}, field_o: 'Integer'):
+    def a_op(field_m: {'GCSPath': {'path_type': 'file', 'file_type':'tsv'}}, field_o: Integer()):
```

Comment:
Are both `'Integer'` and `Integer()` valid?

Reply:
Yes. But, Integer class contains the schema while 'Integer' does not.

Reply:
I see.
Is it OK if the "schema-less" 'Integer' will also be checked by the UI?